nixos/modules/hardware/video/hidpi.nix

```diff
@@ -12,11 +12,11 @@ with lib;
     boot.loader.systemd-boot.consoleMode = mkDefault "1";
 
 
-    # Grayscale anti-aliasing for fonts
-    fonts.fontconfig.antialias = mkDefault true;
-    fonts.fontconfig.subpixel = {
-      rgba = mkDefault "none";
-      lcdfilter = mkDefault "none";
+    # Disable font anti-aliasing, hinting, and sub-pixel rendering by default
+    fonts.fontconfig = {
+      antialias = mkDefault false;
```

Comment:
nit: there seems to be a space at the left of this line

Reply:
Which line?  I'm not seeing in my usual editor (Emacs with `nix-mode`, enabled `whitespace-mode` to check) nor with `nano` in its default config... but `grep` seems to report some?  ^^'

```shell
$ grep -E '\w$' -- nixos/modules/hardware/video/hidpi.nix
    # Needed when typing in passwords for full disk encryption
    # Disable font anti-aliasing, hinting, and sub-pixel rendering by default
    # TODO Find reasonable defaults X11 & wayland
```

Using `sed` to get rid of it also removed the last character of each of those lines, so I suspect there might be some "funny" codepoints in there, I'll check that shortly  >_>'

Reply:
That doesn't seem to be Unicode shenanigans: checking the change in that first line with an hex viewer, only the letter `n` (0x6E) is gone, the linebreak is still a single `\n`, then comes the next line's indenting spaces (0x20)

I'm feeling hopelessly confused, so I'll leave it as is and check again tomorrow, but feel free to push a commit to my PR branch fixing the nit.

Reply:
"Mystery" solved: I am being very sleepy and `\w` is not whitespace but word boundaries, and of course POSIX regexp considers those to be characters somehow

... and `rg` confirms there's not trailing whitespace  :sweat_smile: